falcon_multiqc/commands/remove.py

```diff
@@ -0,0 +1,45 @@
+import click
+from database.crud import session_scope
+from sqlalchemy.orm import Query
+from .query import print_overview
+from database.models import Base, Batch, Cohort
+
+# Command for removing entries from database.
+# --overview to see all the cohorts with there respective batches and number of samples
+# --cohort option requires one argument 'cohortID' - removes all entries associated with that cohort throughout db
+# --batch option requires two arguments 'cohortID and Batch' - removes all entries associated with that batch throughout db
+# Note: although you can use multiple --batch or --cohort options in one command, if both are used together, only --cohort will commit
+
+@click.option("-c", "--cohort", multiple=True, required=False, help="Which cohort to remove from the database. E.g. <MGRB>")
+@click.option ("-b", "--batch", multiple=True, type=(str, str), required=False, help="Which batch you want to remove from the database. E.g. <MGRB BAB>")
+@click.option("-ov", "--overview", is_flag=True, required=False, help="Prints an overview of the number of samples in each batch/cohort.")
```

Comment:
Have you tested this out along with the other commands? Also a really small and unnecessary thing but personally not a big fan of `-ov` but `-o` is confusing and I can't think of anything else 😝

Reply:
lol yeah I didn't like it too, I'll just scrap the short hand haha